app/assets/stylesheets/article-show.scss

```diff
@@ -31,6 +31,97 @@
   height: 10px;
 }
 
+.mod-actions-menu {
+  position: fixed;
+  top: var(--header-height);
+  right: 0;
+  height: calc(100vh - 56px);
```

Comment:
If `56px` is coming from header's height you could do this:

```
height: calc(100vh - var(--header-height));
```

...or your can do:

```
top: var(--header-height);
bottom: 0;
//drop the height.
```

Reply:
Hmm interesting, will test this out